Added a theme for Elementary OS

bartzaalberg opened the pull request:
Added a qss file which fits nicely into the template elementary OS

ManuelSchneid3r commented on data/themes/Elementary.qss:
```diff
@@ -0,0 +1,108 @@
+/*
+ * author: Manuel Schneider <https://github.com/ManuelSchneid3r>
```
Please change the author.

ManuelSchneid3r commented:
Please send a screenie with visible results and actions. (shutter is good to use for demo)

ManuelSchneid3r commented:
like this

![bildschirmfoto vom 2017-02-27 11-35-16](https://cloud.githubusercontent.com/assets/3252315/23358097/e4058b88-fce0-11e6-8df5-63bbc23d1b6e.png)

bartzaalberg commented:
Here you go!
![elementaryosalberttheme](https://cloud.githubusercontent.com/assets/3491721/23358428/33c984f2-fce2-11e6-8406-dab957c55197.png)

bartzaalberg commented:
Oh damn 0.o how do you get those actions? i don't have em.

ManuelSchneid3r commented:
You should read the [docs](https://albertlauncher.github.io/docs/) :D

bartzaalberg commented:
Oh found it! Sorry

bartzaalberg commented:
![selection_002](https://cloud.githubusercontent.com/assets/3491721/23358772/99bb2508-fce3-11e6-932c-ba1025353c63.png)

ManuelSchneid3r commented:
Do you really want to have that much padding?

bartzaalberg commented:
Yes. Find it more in the style of Elementary. They use alot of padding as you can see here.
![selection_003](https://cloud.githubusercontent.com/assets/3491721/23358925/1eb51444-fce4-11e6-8caa-ef9ebceed4ac.png)

bartzaalberg commented:
Changed text size and some paddings. Also Squashed all the commits together. This is how it looks now.

![selection_005](https://cloud.githubusercontent.com/assets/3491721/23512049/9572aa74-ff5f-11e6-907d-eefe19c307e9.png)

ManuelSchneid3r commented:
The gear looks weird

bartzaalberg commented:
This better?

![selection_007](https://cloud.githubusercontent.com/assets/3491721/23607075/be1be1be-0263-11e7-8abf-7cd9bb1700c7.png)

owcz commented:
This should use the elementary system typeface too, no? You can also steal most of the styling from [their website](https://elementary.io/)

More examples of text boxes at https://elementary.io/docs/human-interface-guidelines

bartzaalberg commented:
Also, alot of icons in elementary aren't working for me in Albert. Is this a bug? Or something elementary should fix.

bartzaalberg commented:
And is this a possibility?
![image](https://cloud.githubusercontent.com/assets/3491721/23665228/2c77a5e0-0358-11e7-8f58-6cae6beadeea.png)

ManuelSchneid3r commented:
What you want is a completely new skin. You will be able to do this in future, although its a laod of more work. But atm you can only theme the frontend.

ManuelSchneid3r commented:
Please post a new screenie of the full theme with actionslist.

bartzaalberg commented:
Fixed a final bug. This is probably how far i can take it with the current options
![screenshot from 2017-03-21 14-50-39](https://cloud.githubusercontent.com/assets/3491721/24150391/e647a856-0e45-11e7-9c4a-eb94526d0ffd.png)

ManuelSchneid3r commented:
Albert complains

```
[15:31:02] [DEBG] Unknown property fill
[15:31:02] [DEBG] Unknown property src
```
